MacPass/MPAutotypeDaemon.m

```diff
@@ -267,18 +272,7 @@ - (void)_performAutotypeForEntry:(KPKEntry *)entryOrNil {
   if(self.matchSelectionWindow) {
     return; // we present the match selection window, just return
   }
-  if(!entryOrNil) {
-    NSUserNotification *notification = [[NSUserNotification alloc] init];
-    notification.title = NSApp.applicationName;
-    notification.userInfo = @{ MPUserNotificationTypeKey: MPUserNotificationTypeAutotypeFeedback };
-    if(context) {
-      notification.informativeText = [NSString stringWithFormat:NSLocalizedString(@"AUTOTYPE_OVERLAY_SINGLE_MATCH_FOR_%@", "Notification: Autotype found a single match for %@ (string placeholder)."), self.targetWindowTitle];
-    }
-    else {
-      notification.informativeText = [NSString stringWithFormat:NSLocalizedString(@"AUTOTYPE_OVERLAY_NO_MATCH_FOR_%@", "Noticiation: Autotype failed to find a match for %@ (string placeholder)"), self.targetWindowTitle];
-    }
-    [NSUserNotificationCenter.defaultUserNotificationCenter deliverNotification:notification];
-  }
+
```

Comment:
The notification should only be showed when the user has no other feedback. Removing it here and moving it to the ```_performAutotypeForContext:``` will show it every time.

Reply:
Like the previous question, the result of each fill should tell the user